.gitlab-ci.yml

```diff
@@ -0,0 +1,47 @@
+################################################################################
+# VIKUNJA_CXX                             : {g++, clang++}
+#   [g++]                                 : {5, 6, 7, 8, 9} <list>
+#   [clang++]                             : {4.0, 5.0, 6.0, 7, 8, 9, 10} <list>
+# VIKUNJA_BOOST_VERSIONS                  : {1.65.1, 1.66.0, 1.67.0, 1.68.0, 1.69.0, 1.70.0, 1.71.0, 1.72.0, 1.73.0} <list>
+# VIKUNJA_BUILD_TYPE                      : {Debug, Release}
+# VIKUNJA_CMAKE_ARGS                      : <string>
+include:
+  - local: '/script/compiler_base.yml'
+
+cuda102:
+  image: registry.gitlab.com/hzdr/crp/alpaka-group-container/alpaka-ci-cuda102-gcc:1.3
+  variables:
+    VIKUNJA_BOOST_VERSIONS: "1.65.1 1.66.0 1.67.0 1.68.0 1.69.0 1.70.0 1.71.0 1.72.0 1.73.0"
+  extends: .base_cuda
+
+cuda112:
+  image: registry.gitlab.com/hzdr/crp/alpaka-group-container/alpaka-ci-cuda112-gcc:1.3
+  variables:
+    VIKUNJA_BOOST_VERSIONS: "1.65.1 1.66.0 1.67.0 1.68.0 1.69.0 1.70.0 1.71.0 1.72.0 1.73.0"
+  extends: .base_cuda
+
+gcc:
+  variables:
+    VIKUNJA_CXX: "g++-5 g++-6 g++-7 g++-8 g++-9"
+    VIKUNJA_BOOST_VERSIONS: "1.65.1 1.75.0"
+  extends: .base_gcc
+
+clang:
+  variables:
+    VIKUNJA_CXX: "clang++-5.0 clang++-6.0 clang++-7 clang++-8 clang++-9 clang++-10 clang++-11"
+    VIKUNJA_BOOST_VERSIONS: "1.65.1 1.75.0"
+  extends: .base_clang
+
+# cudaClang101:
```

Comment:
Why is this commented out?

Reply:
Because I want to shift the Clang-CUDA support to later. But to be honest, I can test it now :thinking:

Reply:
It does not compile. I will postpone it.